documentation/src/main/asciidoc/topics/code_examples/LibraryInitializerImpl.java

```diff
@@ -0,0 +1,48 @@
+/*
+ Generated by org.infinispan.protostream.annotations.impl.processor.AutoProtoSchemaBuilderAnnotationProcessor
+ for class org.infinispan.commons.marshall.LibraryInitializer
+ annotated with @org.infinispan.protostream.annotations.AutoProtoSchemaBuilder(dependsOn=, service=false, autoImportClasses=false, excludeClasses=, includeClasses=org.infinispan.commons.marshall.Library.Book,org.infinispan.commons.marshall.Library.Author, basePackages={}, value={}, schemaPackageName="book_sample", schemaFilePath="proto/", schemaFileName="library.proto", className="")
+ */
+
+package org.infinispan.commons.marshall;
+
+/**
+ * WARNING: Generated code!
+ */
+@javax.annotation.Generated(value = "org.infinispan.protostream.annotations.impl.processor.AutoProtoSchemaBuilderAnnotationProcessor",
+      comments = "Please do not edit this file!")
+@org.infinispan.protostream.annotations.impl.OriginatingClasses({
+      "org.infinispan.commons.marshall.Library.Author",
+      "org.infinispan.commons.marshall.Library.Book"
+})
+/*@org.infinispan.protostream.annotations.AutoProtoSchemaBuilder(
```

Comment:
comment

Reply:
This is generated code, I'm just copy and pasting to show what is generated.